android/src/main/java/com/rt2zz/reactnativecontacts/ContactsProvider.java

```diff
@@ -304,12 +304,20 @@ public WritableArray getContacts() {
         return contacts;
     }
 
+    private Boolean cursorMoveToNext(Cursor cursor) {
+        try {
+            return cursor.moveToNext();
+        } catch(RuntimeException error) {
+            return false;
```

Comment:
what will happen if the information is too big? will the contact not be returned altogether?

Reply:
If a `RuntimeException` is thrown, `cursorMoveToNext` will return `false`, which cause any subsequent items from being parsed (if used in a while loop).

Yes, this will result in a bug, but at least it will not result in a crash. I added some error logging if the exception is thrown.